docs/architecture/governance.md

```diff
@@ -1,50 +1,86 @@
 ---
-title: Governance
+title: Governance and self-amendment
 last_update:
-  date: 3 July 2023
+  date: 22 February 2024
 ---
 
-## What is Self Amendment?
+Tezos incorporates a built-in, on-chain mechanism for proposing, selecting, testing, and activating protocol upgrades without the need to hard fork.
+This mechanism makes Tezos a self-amending blockchain and allows any user to propose changes.
 
-Tezos is a self-amending blockchain network that incorporates an on-chain mechanism for proposing, selecting, testing, and activating protocol upgrades without the need to hard fork.
+All of these self-amendment features, including the scheduling and operation of each amendment period and the voting processes, are automatic and handled by the protocol without human intervention.
 
-## How Does It Work?
+## Amendment periods
 
-The self-amendment process is split into 5 periods: Proposal Period, Exploration Vote Period, Cooldown Period, Promotion Vote Period, and Adoption Period. Each of these periods lasts five baking cycles (i.e. 40,960 blocks at 30-second intervals or roughly 14 days, 5 hours), comprising roughly 2 months and 10 days.
+The self-amendment process is split into 5 periods:
 
-Should there be any failure to proceed for a period, the whole process reverts to the Proposal Period, effectively restarting the whole process.
+- Proposal period: Users propose changes to the Tezos protocol and vote on a single proposal to explore; the top-voted proposal moves to the next period
+- Exploration Vote period: Users vote whether to consider the top-voted proposal
+- Cooldown period: If the proposal gets enough votes in the Exploration Vote period, users have the length of the Cooldown period to review and test the new protocol
```

Comment:
```suggestion
- Cooldown period: If a proposal gets is selected in the Exploration period, a Cooldown period starts before the final election is made. The community can continue testing the new protocol proposal, and preparing their infrastructure, before the final decision is made. 
```
Reworded a bit.

The original explanation lingers from when the cooldown period actually forked a "testchain" (not a test network) on the side of Tezos mainnet, using the selected protocol. This feature was dropped, as indeed the period was too short, and we just renamed it to Cooldown.

Still, the "users has the length of the Cooldown period to review and test the new protocol" still reads weird. Tesing the protocol started well in advance, in protocol testnets which normally start w/ the Proposal period... and in periodict test networks like weeklynet.